syntax = "proto2";

import "Event.proto";

package jungi.mobgtw.proto;
option optimize_for = LITE_RUNTIME;

message CallEvents {
  repeated Event events = 1;
}
